mirrulations-work-generator/src/mirrgen/work_generator.py

```diff
@@ -27,27 +34,17 @@ def download(self, endpoint):
 
 
 if __name__ == '__main__':
-
-    def is_redis_available(database):
-        try:
-            # return fakeredis.FakeRedis('localhost').ping() # Not sure if the database is at localhost or 'redis'
-            return database.ping() # Not sure if the database is at localhost or 'redis'
-        except (ConnectionRefusedError, redis.BusyLoadingError) as error:
-            return False
-
     # I wrapped the code in a function to avoid pylint errors
     # about shadowing api and job_queue
     def generate_work():
         dotenv.load_dotenv()
         api = RegulationsAPI(os.getenv('API_KEY'))
 
         database = redis.Redis('redis')
-        
         while not is_redis_available(database):
             print("error")
```

Comment:
We probably don't want to print "error" here. Might want to print "Loading" or something similar, since the errors are caught.